packages/rpc-graphql/src/__tests__/account-test.ts

```diff
@@ -288,37 +293,36 @@ describe('account', () => {
             expect(result).toMatchObject({
                 data: {
                     account: {
+                        address: 'CcYNb7WqpjaMrNr7B1mapaNfWctZRH7LyAjWRLBGt1Fk',
                         data: 'KLUv/QBYSQAAdGVzdCBkYXRh',
                     },
                 },
             });
         });
     });
     describe('specific account type queries', () => {
-        it('can get a mint account', async () => {
+        it('can get a nonce account', async () => {
             expect.assertions(1);
-            // See scripts/fixtures/spl-token-mint-account.json
+            // See scripts/fixtures/nonce-account.json
             const variableValues = {
-                address: 'Gh9ZwEmdLJ8DscKNTkTqPbNwLNNBjuSzaG9Vp2KGtKJr',
+                address: 'AiZExP8mK4RxDozh4r57knvqSZgkz86HrzPAMx61XMqU',
             };
             const source = /* GraphQL */ `
                 query testQuery($address: String!) {
                     account(address: $address) {
-                        ... on MintAccount {
-                            data {
-                                decimals
-                                isInitialized
-                                mintAuthority {
-                                    address
-                                    lamports
-                                }
-                                supply
+                        address
+                        lamports
+                        ... on NonceAccount {
+                            authority {
+                                address
                             }
-                            meta {
-                                program
-                                space
-                                type
+                            blockhash
+                            feeCalculator {
+                                lamportsPerSignature
                             }
```

Comment:
Tangent; this is hella deprecated right? Should we do something else here?

Reply:
Hah, I didn't check if it was deprecated 😅

Something different, like, take it out completely?

Reply:
Take it out if there's no non-deprecated way to get at it, or flatten it down to the same level as `blockhash` and give it a good name.

Reply:
<img width="956" alt="image" src="https://github.com/solana-labs/solana-web3.js/assets/13243/c794a152-fb5c-4add-9df0-5950662bbd2b">